pubspec.yaml

```diff
@@ -12,7 +12,7 @@ environment:
 dependencies:
   args: '>=0.10.0 <0.13.0'
   http_server: '>=0.9.0 <0.10.0'
-  quiver: '>=0.18.0 <0.21.0'
+  quiver: '>=0.18.0 <0.23.0'
```

Comment:
`quiver` is on `0.21.0+3` right now, so we should limit this to <0.22.0. You are jumping the gun :)